modules/manually-restoring-cluster-etcd-backup.adoc

```diff
@@ -0,0 +1,299 @@
+// Module included in the following assemblies:
+//
+// * disaster_recovery/scenario-2-manually-restoring-cluster-etcd-backup.adoc
+// * post_installation_configuration/cluster-tasks.adoc
+
+
+:_content-type: PROCEDURE
+[id="manually-restoring-cluster-etcd-backup_{context}"]
+= Restoring a cluster manually from an etcd backup
+
+You can use a saved etcd backup to restore a previous cluster state or restore a cluster that has lost the majority of control plane hosts. The procedure described below only uses the already existing control plane nodes.
+
+If your cluster uses a control plane machine set, see "Troubleshooting the control plane machine set" for a simpler etcd recovery procedure.
+
+When you restore your cluster, you must use an etcd backup that was taken from the same z-stream release. For example, an {product-title} 4.7.2 cluster must use an etcd backup that was taken from 4.7.2.
+
+[NOTE]
+====
+The standard restore procedure requires the complete recreation of 2 control plane nodes, which might be complex for clusters installed with UPI, since an UPI installation does not create any MachineSet for the control plane nodes.  It also uses the script `/usr/local/bin/cluster-restore.sh`, which starts a new single-member etcd cluster, then scales it to 3 members. The procedure described in this guide, instead, directly starts a 3-members etcd cluster.
+====
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* A healthy control plane host to use as the recovery host.
+* SSH access to _all_ control plane hosts.
+* All control plane hosts can be accessed through SSH using the root user. 
+* A backup directory containing both the etcd snapshot and the resources for the static pods from the same backup. The file names in the directory must be in the following formats: `snapshot_<datetimestamp>.db` and `static_kuberesources_<datetimestamp>.tar.gz`.
+
+.Procedure
+
+. Select a control plane host to use as the recovery host. This is the host that you will run the restore operation on.
+
+. Use SSH to connect to each of the control plane nodes, including the recovery host.
++
+The Kubernetes API server becomes inaccessible after the restore process starts, so you cannot access the control plane nodes. For this reason, it is recommended to use a SSH connection for each control plane host you are accessing in a separate terminal.
++
+[IMPORTANT]
+====
+If you do not complete this step, you will not be able to access the control plane hosts to complete the restore procedure, and you will be unable to recover your cluster from this state.
+====
+
+. Copy the etcd backup directory to the recovery control plane host.
```

Comment:
```suggestion
. Copy the etcd backup directory to a control plane host.
```

Reply:
Done

Reply:
Done the changes as per the latest comment below.